tensorrt_llm/_torch/attention_backend/sparse/kernels/common.py

```
@@ -0,0 +1,935 @@
+import math
+
+import torch
+import triton
+import triton.language as tl
+import triton.language.core as core
+from triton.language.standard import _log2, sum, zeros_like
```

Ruff (0.15.10)

[error] Import `sum` is shadowing a Python builtin

(A004)